### PR TITLE
add sendbutton to types.d.ts

### PR DESCRIPTION
**Pull request by jaapbakker88**
# Submit a pull request

Regarding #97 

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: types/index.d.ts [hunk 1]
```diff
@@ -286,6 +286,9 @@ export interface MessageInputProps {
   /** The component handling how the input is rendered */
   Input?: React.ElementType<MessageInputUIComponentProps>;
 
+  /** Change the SendButton component */
+  SendButton?: React.ElementType<any>;
```
Comment (vishalnarkhede) [inline types/index.d.ts]: Instead of `any`, we should create an interface called `SendButtonProps` ... and assign it here!!

**Comment by damianprzygodzki** [comment]
But for god sake! How i should obtain message for send button? What's the idea of requiring message in SendButton?

**Comment by tbarbugli** [comment]
@jaapbakker88 the message is passed to the button component's `sendMessage` function right? perhaps we should include an example of a custom button in our docs

@damianprzygodzki what's your use-case for replacing the SendButton component?

**Comment by damianprzygodzki** [comment]
I just want to place another component in that place to MessageInput as prop (as documented).

MessageInput already have that message, so i suppose SendButton is only responsible for 'clicking'.

**Comment by jaapbakker88** [comment]
@tbarbugli I'll add an example of how to use this in our docs. @damianprzygodzki the sendMessage prop just accepts a function that sends the message. Could have been more conveniently  named I think.

**Pull request merged**